### PR TITLE
SER-55, SER-56: Update to do and served views with new columns

### PR DESCRIPTION
**Pull request by acsauk**

### CODE REVIEW

File: serve-web/features/04-cases.feature [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@acs
```
Comment (Gugandeep) [inline serve-web/features/04-cases.feature:1]: Think this was meant to be removed?

**Review by Gugandeep — COMMENTED** [review]
Looks good to go.

Just a tag that was accidently left in I think.

File: serve-web/src/Migrations/Version20210623150012.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20210623150012 extends AbstractMigration
+{
+    public function getDescription() : string
```
Comment (coderste) [inline serve-web/src/Migrations/Version20210623150012.php:12]: Don't mean to be a pain the `:` after the parenthesis have spaces but in other files they don't. 

Looking at Digideps which has the formatting it looks like these spaces shouldn't be here

**Review by coderste — CHANGES_REQUESTED** [review]

**Review by coderste — APPROVED** [review]

File: serve-web/features/04-cases.feature (hunk 1 above)
Comment (acsauk) [reply serve-web/features/04-cases.feature]: Actioned ✅

File: serve-web/src/Migrations/Version20210623150012.php (hunk 2 above)
Comment (acsauk) [reply serve-web/src/Migrations/Version20210623150012.php]: Actioned ✅ 

As per Slack - I'll add some pre-commit config to the project on a new PR and run PHP CS fixer on all the files.

**Pull request merged**